rpcclient/infrastructure.go

```diff
@@ -776,8 +776,10 @@ func (c *Client) handleSendPostMessage(jReq *jsonRequest) {
 
 	tries := 10
 	for i := 0; i < tries; i++ {
+		var httpReq *http.Request
+
 		bodyReader := bytes.NewReader(jReq.marshalledJSON)
-		httpReq, err := http.NewRequest("POST", url, bodyReader)
+		httpReq, err = http.NewRequest("POST", url, bodyReader)
```

Comment:
When you say the error didn't leave the scope, you mean this `err` here was somehow dropped in the channel send below? Still trying to wrap my head around the fix here...

Reply:
This `:=` overrides the `err` defined at line 772, making `err` outside this scope always `nil`.